src/components/Login/Login.module.scss

```diff
@@ -0,0 +1,38 @@
+@import 'variables.scss';
+
+.c-login {
+  height: 100%;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  &__container {
+    padding: $space-xs $space-s;
+    margin: auto;
+
+    max-width: fit-content;
+    max-height: fit-content;
+
+    background: $color-gray;
+    color: $color-black;
+
+    border-radius: 5%;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+
+    &__span {
```

Comment:
This shouldn't be named span since span is name of html tag and it is not really meaningful. I made suggestion for this in component code so you can look it up.